D_controllers.md

```diff
@@ -235,22 +255,22 @@ What if we want to plug render, but only some of our actions should actually cal
 
 By default, the render plug takes over all rendering for all actions in a controller. Once we plug render, if we try to use any rendering-style function in any action - `text/2`, `json/2`, `html/2`, `render/2` or even `redirect/2`- we will get errors if that action is dispatched to. Even if the application appears to do the right thing, the server will be throwing errors.
 
-Phoenix offers a solution to this by letting us specify which actions `plug :render` should be applied to. If we only wanted `plug :render` to work on the `index` and `show` actions, we could do this.
+Phoenix offers a solution to this by using guard expressions to let us specify which actions `plug :render` should be applied to. If we only wanted `plug :render` to work on the `index` and `show` actions, we could do this.
```

Comment:
I think I prefer the original in this case. Using the term "guard expression" might lead to confusion because it is so similar to "guard clause", which is quite different.